integration-tests/src/test/resources/scripts/restart.sh

```diff
@@ -0,0 +1,3 @@
+sudo systemctl stop constellation && \
+  sudo rm -rf /home/admin/constellation/tmp && \
+  sudo systemctl start constellation
```

Comment:
what about logs? do you want to store them between restarts? we used to either remove or move logs to have clean logs every restart - you don't need to find in logs when restart happened (i.e. when searching for errors in the current run)

Reply:
I guess I can rename the logs, but I would rather not remove them entierly.